e2e-tests/cypress/integration/eligibility-decision-date-page-components.feature

```diff
@@ -52,4 +52,4 @@ Feature: The Decision Date page has the right structure
 
       Examples:
           | Day   | Month   | Year   |
-          | "01"  | "11"    | "2020" |
+          | "24"  | "01"    | "2021" |
```

Comment:
Has this changed, or is the example just being updated?

Reply:
I added the change because it triggered a test failure due to the 12 week period elapsing. I think the whole feature file is now wip.